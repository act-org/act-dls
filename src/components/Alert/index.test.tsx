/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import { Alert } from '.';

describe('Alert', () => {
  it('ACT theme matches the snapshot', () => {
    const { container } = render(
      <Alert text="Some alert description goes here." title="Title Here" />,
      'ACT',
    );
    expect(container).toMatchSnapshot();
  });

  it('ACT_ET theme matches the snapshot', () => {
    const { container } = render(
      <Alert text="Some alert description goes here." title="Title Here" />,
      'ACT_ET',
    );
    expect(container).toMatchSnapshot();
  });
});
