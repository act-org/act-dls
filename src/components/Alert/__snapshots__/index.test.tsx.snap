// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert ACT theme matches the snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiAlert-root MuiAlert-standardInfo MuiPaper-elevation0"
    role="alert"
    text="Some alert description goes here."
  >
    <div
      class="MuiAlert-icon"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      class="MuiAlert-message"
    >
      <div
        class="MuiTypography-root MuiAlertTitle-root MuiTypography-body1 MuiTypography-gutterBottom"
      >
        Title Here
      </div>
    </div>
  </div>
</div>
`;

exports[`Alert ACT_ET theme matches the snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiAlert-root MuiAlert-standardSuccess MuiPaper-elevation0"
    role="alert"
    text="Some alert description goes here."
  >
    <div
      class="MuiAlert-icon"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M20,12A8,8 0 0,1 12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4C12.76,4 13.5,4.11 14.2, 4.31L15.77,2.74C14.61,2.26 13.34,2 12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0, 0 22,12M7.91,10.08L6.5,11.5L11,16L21,6L19.59,4.58L11,13.17L7.91,10.08Z"
        />
      </svg>
    </div>
    <div
      class="MuiAlert-message"
    >
      <div
        class="MuiTypography-root MuiAlertTitle-root MuiTypography-body1 MuiTypography-gutterBottom"
      >
        Title Here
      </div>
    </div>
  </div>
</div>
`;
